### PR TITLE
feat: Pager component: adding first and last and adding addAttributes

### PR DESCRIPTION
**Pull request by mndonx**
**Summary**

This PR improves the Pager component for Compound.

- Adds First and Last option to Pager (this is regularly seen in Drupal sites)
- Adds `add_attributes` to template (to make it easier to extend them when we implement them in other components)
- Renamed `pager-ellipses.yml` to `pager-next-ellipses.yml` to match `prev` example for consistency
- Hid the ellipses on the basic pager example (it didn't seem to make sense to keep them in without the arrows there too.)

**How to review this PR**
- Check out `feat-pager` branch
- `npm run develop`
- Navigate to:
  http://localhost:6006/?path=/story/molecules-menus-pager--basic
  http://localhost:6006/?path=/story/molecules-menus-pager--with-next
  http://localhost:6006/?path=/story/molecules-menus-pager--with-both
  http://localhost:6006/?path=/story/molecules-menus-pager--with-previous
  http://localhost:6006/?path=/story/molecules-menus-pager--with-first-and-last
- Verify that changes above make sense and the code looks good

### CODE REVIEW

File: components/02-molecules/pager/pager.twig [hunk 1]
```diff
@@ -31,9 +31,22 @@
 #}
 {% set pager__base_class = 'pager' %}
 
+{% set pager__attributes = pager__attributes|default({
+  'class': bem(pager__base_class, pager__modifiers, pager__blockname),
+}) %}
+
 {% if items %}
-  <nav {{ bem(pager__base_class) }} aria-label="{{ pagination_id|default('Pagination'|t) }}">
+  <nav {{ add_attributes(pager__attributes) }} aria-label="{{ pagination_id|default('Pagination'|t) }}">
```
[the remark this answers is not in the document]
Comment (callinmullaney) [reply components/02-molecules/pager/pager.twig:39]: @mndonx Just wanted to confirm Joe's feedback is valid and `aria-label` should be moved to the `add_attributes` function

**Comment by mndonx** [comment]
@callinmullaney @joetower Thanks! I've moved the aria-label into the pager__attributes.

**Review by joetower — APPROVED** [review]
@mndonx This looks great! :clap: 

I think it's ready to review in our L10.

**Review by ModulesUnraveled — CHANGES_REQUESTED** [review]
- [ ] Let's use array numbers to get rid of `0`s.
- [ ] Make the current page more obvious, and add a `current` to each example.
- [ ] Add `aria-current="page"` to the current link.
- [ ] Please check to see what the Drupal text is for "first" and "last"
- [ ] You can remove all of the empty `attributes` from yaml files
- [ ] Add `role="navigation"` to the nav

**Comment by mndonx** [comment]
@ModulesUnraveled 

> * [x]  Let's use array numbers to get rid of `0`s.
> * [x]  Make the current page more obvious, and add a `current` to each example.
> * [x]  Add `aria-current="page"` to the current link.
> * [x]  Please check to see what the Drupal text is for "first" and "last"
> * [x]  You can remove all of the empty `attributes` from yaml files
> * [x]  Add `role="navigation"` to the nav

Also:
- I removed yml references to `pager-uuid` -- I can't find this anywhere, and not sure why it was there. 
- I also stripped down the theming of the next/prev buttons. I feel like the css arrows are more of a design choice, and we're trying to avoid that here. 

Let me know what you think!
Amanda

**Comment by mndonx** [comment]
Note to self to adjust item padding at small widths

**Review by mcortes19 — APPROVED** [review]

**Review by mariannuar — APPROVED** [review]
LGTM!

**Pull request merged**